src/overlays/actors/ovl_Bg_Spot16_Bombstone/z_bg_spot16_bombstone.c

```diff
@@ -9,7 +10,96 @@ void BgSpot16Bombstone_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot16Bombstone_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot16Bombstone_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_808B5A94(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B04(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B6C(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5B58(BgSpot16Bombstone* this);
+void func_808B5950(BgSpot16Bombstone* this, GlobalContext* globalCtx);
+void func_808B5934(BgSpot16Bombstone* this);
+void func_808B5AF0(BgSpot16Bombstone* this);
+void func_808B5A78(BgSpot16Bombstone* this);
+
+extern Gfx D_06000C20[];
+extern Gfx D_060009E0[];
+
+typedef struct D_808B5DD8Struct {
+    s16 speedXZ;
+    s16 speedY;
+    s16 scale;
+    s16 unk_6;
+    s16 unk_8;
+    s16 unk_A;
+    s16 unk_C;
+    Vec3s rotation;
+} D_808B5DD8Struct; // size = 0x14
+
+typedef struct D_808B5EB0Struct {
+    s16 unk_0;
+    s16 unk_2;
+    s16 unk_4;
+    s16 unk_6;
+    s16 scale;
+    s16 scaleStep;
+    s16 life;
+} D_808B5EB0Struct; // size = 0xE
+
+static EnBombf* sPlayerBomb = NULL;
+
+static s16 sTimer = 0;
+
+D_808B5DD8Struct D_808B5DD8[] = { { 0x0008, 0x0004, 0x0046, 0x07D0, 0xFCE0, 0x0000, 0x0064, 0x0000, 0x0000, 0x0000 },
+                                  { 0x0006, 0x0003, 0x0032, 0x00C8, 0x0A28, 0xC350, 0x005A, 0x0000, 0x0000, 0x0000 },
+                                  { 0x0005, 0x0003, 0x0028, 0xF63C, 0x0190, 0x30B0, 0x0032, 0x0000, 0x0000, 0x0000 },
+                                  { 0x0003, 0x0001, 0x003C, 0x0258, 0xFF9C, 0xAFC8, 0x0032, 0x0000, 0x0000, 0x0000 },
+                                  { 0x0003, 0x0001, 0x0028, 0xF2B8, 0xFF9C, 0x6590, 0x001E, 0x0000, 0x0000, 0x0000 },
+                                  { 0x0006, 0x0009, 0x0028, 0x0000, 0x0BB8, 0xD8F0, 0x001E, 0x0000, 0x0000, 0x0000 } };
+
+static ColliderJntSphItemInit sJntSphItemsInit[] = {
+    {
+        { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x4FC1FFF6, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+        { 0, { { 0, 50, 0 }, 288 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK12, 0x00, 0x0D, 0x39, 0x20, COLSHAPE_JNTSPH },
+    1,
+    sJntSphItemsInit,
+};
+
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x00, 0x00, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000008, 0x00, 0x00 }, 0x00, 0x01, 0x00 },
+    { 190, 80, 0, { 10, 0, 50 } },
+};
+
+D_808B5EB0Struct D_808B5EB0[] = { { 0x0000, 0x000A, 0x003C, 0xFFF6, 0x0104, 0x01E0, 0x0007 },
```

Comment:
you can make this `static`, and if you add a trailing comma and run the formatter it'll get rid of the big padding
`{ 0x0014, 0x0050, 0x0032, 0x0000, 0x0096, 0x00C8, 0x0008 } };` (last line)
-> `{ 0x0014, 0x0050, 0x0032, 0x0000, 0x0096, 0x00C8, 0x0008 }, };`